### ISSUE TITLE
Localize plugin labels

### ISSUE THREAD

**Issue by doronrosenberg** [maqetta/maqetta#2720]
All the plugins .js files have hard coded strings

**Comment by aerwin** [comment]
--> prio from not set to med

**Comment by JonFerraiolo** [comment]
-> @jhpedemonte (wasn't assigned to anyone) - maybe push to M8?

**Comment by jhpedemonte** [comment]
-> M8

**Comment by JonFerraiolo** [comment]
I don't think this will be all that difficult to do. Somehow add a field which points to a localization file, and then replace all of the hard-coded strings with indices into the localization file.

**Comment by jhpedemonte** [comment]
Raising to Prio High for M8, so we take a look at it earlier, rather than later.

**Comment by JonFerraiolo** [comment]
Just added #3101

**Comment by JonFerraiolo** [comment]
-> M8 Prio-Med

**Comment by jhpedemonte** [comment]
-> M9
